Okta.AspNet/OpenIdConnectAuthenticationOptionsBuilder.cs

```diff
@@ -5,67 +5,147 @@
 
 using System.Linq;
 using System.Net.Http;
+using System.Security.Claims;
+using System.Threading.Tasks;
 using Microsoft.IdentityModel.Protocols;
 using Microsoft.IdentityModel.Protocols.OpenIdConnect;
 using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.Notifications;
 using Microsoft.Owin.Security.OpenIdConnect;
 using Okta.AspNet.Abstractions;
 
 namespace Okta.AspNet
 {
     public class OpenIdConnectAuthenticationOptionsBuilder
     {
+        private OktaMvcOptions _oktaMvcOptions;
+        private IUserInformationProvider _userInformationProvider;
+        private string _issuer;
+        private ConfigurationManager<OpenIdConnectConfiguration> _configurationManager;
+        private HttpClient _httpClient;
```

Comment:
nit: Consider making all of these `readonly`. This class is immutable and its a nice touch when the reader can immediately tell that just from looking at the declarations.